app/javascript/components/menus/edit/edit-menu-container.vue

```diff
@@ -0,0 +1,259 @@
+        await updateMenu(this.menuId, updatedMenu);
+        window.location = '/menus';
+      } catch (error) {
+        this.error = error;
+      }
+    },
+    useMenuInfo(menu) {
+      this.menuName = menu.attributes.name;
+      this.portions = menu.attributes.portions;
+      this.portions = menu.id;
+      this.initialRecipes = menu.attributes.menuRecipes.data.map(element => ({
+        'quantity': element.attributes.recipeQuantity,
+        'id': element.attributes.recipe.id,
+        'idMenuRecipe': parseInt(element.id, 10),
+      }));
+      this.addInitialRecipes();
+    },
+    addInitialRecipes() {
+      const ids = this.initialRecipes.map(recipe => recipe.id);
+      this.selectedRecipes = this.recipes.filter(recipe => ids.includes(parseInt(recipe.id, 10)));
+      this.selectedRecipes.forEach(element => {
+        const index = ids.indexOf(parseInt(element.id, 10));
+        element.quantity = this.initialRecipes[index].quantity;
+        element.idMenuRecipe = this.initialRecipes[index].idMenuRecipe;
+      });
+    },
+    getPriceOfSelectedIngredient,
+    addRecipe(recipe) {
+      const defaultQuantity = 1;
+      this.selectedRecipes.push({ ...recipe, quantity: defaultQuantity });
+    },
+    deleteRecipe(recipe) {
+      const indexToRemove = this.selectedRecipes.findIndex((element) =>
+        parseInt(element.id, 10) === parseInt(recipe.id, 10));
+      this.selectedRecipes.splice(indexToRemove, 1);
+      this.deletedRecipes.push(recipe);
+    },
+    increaseQuantity(recipe) {
+      const newValue = recipe.quantity += 1;
+      const indexToUpdate = this.selectedRecipes.findIndex((element) =>
+        parseInt(element.id, 10) === parseInt(recipe.id, 10));
+      this.selectedRecipes.splice(indexToUpdate, 1, { ...recipe, quantity: newValue });
+    },
+    decreaseQuantity(recipe) {
+      const newValue = recipe.quantity -= 1;
+      const indexToUpdate = this.selectedRecipes.findIndex((element) =>
+        parseInt(element.id, 10) === parseInt(recipe.id, 10));
+      this.selectedRecipes.splice(indexToUpdate, 1, { ...recipe, quantity: newValue });
+    },
+    getUpdatedMenu() {
+      const updatedMenu = { name: this.menuName,
+        portions: this.portions };
+      this.addMenuRecipeAttributes(updatedMenu);
+
+      return updatedMenu;
+    },
+    addMenuRecipeAttributes(updatedMenu) {
+      const menuRecipesAttributes = [];
+      this.addNewAndUpdatedRecipes(menuRecipesAttributes);
+      this.addDeletedMenuRecipes(menuRecipesAttributes);
+      updatedMenu.menuRecipesAttributes = menuRecipesAttributes;
+    },
+    addNewAndUpdatedRecipes(menuRecipeAttributes) {
+      for (const menuRecipe of this.selectedRecipes) {
+        const hash = {
+          recipeId: parseInt(menuRecipe.id, 10),
+          recipeQuantity: menuRecipe.quantity,
+        };
+        if (!!menuRecipe.idMenuRecipe) {
+        // id relacion
+          hash.id = menuRecipe.idMenuRecipe;
+          hash._destroy = false;
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected dangling '_' in '_destroy'  no-underscore-dangle